packages/plugin-simple-throttle/test/throttle.test.ts

```diff
@@ -3,11 +3,21 @@ import plugin from '../'
 import Client from '@bugsnag/core/client'
 
 describe('plugin: throttle', () => {
-  it('prevents more than maxEvents being sent', () => {
+  describe('addOnError', () => {
     const payloads = []
     const c = new Client({ apiKey: 'aaaa-aaaa-aaaa-aaaa' }, undefined, [plugin])
+    const mockWarn = jest.fn(message => message)
```

Comment:
I don't think you need to provide a function here

```suggestion
    const mockWarn = jest.fn()
```